### PR TITLE
Add collections to user profile page

### PR DESCRIPTION
**Pull request by Jedeu**
**Changes**

1.  Change user rating from a number to stars
2. Add collections to user profile page
3. Add link that directs you to that collection when you click on it
4. Add generic icon svg for board game collection

user-profile

### CODE REVIEW

**Comment by zachwreed** [comment]
Looks great!

File: client/src/css/UserProfile.css [hunk 1]
```diff
@@ -44,6 +44,76 @@
...
+
+.collections {
+  text-align: center;
+  padding-top: 16px;
+}
+
+.collections .collection-container {
+  border: 1px solid #000;
+  height: 250px;
+  width: 250px;
```
Comment (mobilemelody) [inline client/src/css/UserProfile.css]: I would take this out and just let the Grid component take care of sizing

File: client/src/components/UserProfile.jsx [hunk 2]
```diff
@@ -38,14 +41,40 @@ class _UserProfile extends React.Component {
...
               ))
             }
           </Grid>
         </Grid>
+        <span className="heading">Collections</span>
+        <hr/>
+        <Grid container justify="flex-start" className="collections" spacing={1}>
+          {
+            user.collections.map((collection) => (
+              <Grid item md={3} xs={3}>
```
Comment (mobilemelody) [inline client/src/components/UserProfile.jsx]: Change to `<Grid item md={3} xs={12}>` so they stack on mobile

**Review by mobilemelody — CHANGES_REQUESTED** [review]
Some minor comments regarding responsive layout. 

Also, refreshing doesn't seem to work on the user profile page (just shows a blank screen)?

**Comment by zachwreed** [comment]
> Also, refreshing doesn't seem to work on the user profile page (just shows a blank screen)?

that should be taken care of with my Redux fix.

**Review by mobilemelody — APPROVED** [review]

**Pull request merged**